scripts/commit-filter-check.sh

```diff
@@ -13,14 +13,37 @@ do
   gitmessage=`git log --format=%B -n 1 "$i"`
 
   # fix(commit-filter-check): add commit messages (AEROGEAR-038928990423)
-   messagecheck=`echo $gitmessage | grep -w "feat\|fix\|docs" | grep "(AEROGEAR-" | grep "): "`
+  messagecheck=`echo $gitmessage | grep -w "feat\|fix\|docs"`
```

Comment:
Looking good. It happened to output all checks even though it only failed one. Is this happening for you? Again, I was using `feat: sdasdsadsaas (AEROGEAR-asdsada)`

Reply:
was seeing that issue when using ` instead of " for the string I was passing in. Not it seeing now

Reply:
seems to be working fine for me now, weird.